scripts/testBot/playlistModifier.py

```diff
@@ -103,15 +105,30 @@ def addDisabled(files, args):
                 disabled.append(platEle)
             testCaseName = test[0].find("testCaseName")
             testCaseName.addnext(disabled)
+            updateCopyright(root, args)
             updateFile(file, root)
             updated = True
     return updated
 
-def formatter(files):
+def formatter(files, args):
     for file in files:
         root = etree.parse(file)
+        updateCopyright(root, args)
         updateFile(file, root)
 
+def updateCopyright(root, args):
+    if (args["copyright"] == "true"):
+        cr = root.xpath(f"/comment()")
+        if cr:
+            crtext = cr[0].text
+            find = fr"(.*)(Copyright \(c\) \d\d\d\d, )(\d\d\d\d)(.*)"
```

Comment:
Could we do `(.*)(Copyright \(c\) \d{4}, )(\d{4}) (.*)`?

Reply:
updated, thanks!